### DISCUSSION TITLE
Does anyone updated to Ventura 13.4?

### DISCUSSION THREAD

**Discussion by SashonIT** [osy86/HaC-Mini#1087] [Releases]
Can we use this here for Reporting and collecting any issues with 13.4.

I wont install it now.  Still 13.3.1

I made a second Testsystem....... i will try it there in the next days.

**Comment by SashonIT** [comment]
I did the Update Today.

I can Confirm, that 13.4 is runnin Flawless.  BEfore Install i did a OC Update to OC DEBUG 0.9.2

And InitialMode needs to be set to AUTO ----> under UEFI ----> Output ----> at the Bottom. If you use OCAT.

**Reply by Malumen** [reply]
Can you share example EFI? I am thinking to upgrade to ventura soon.

Both LAN ports work OK in Ventura?
